src/rez/plugin_managers.py

```diff
@@ -235,9 +239,9 @@ def config_schema(self):
                 deep_update(d, d_)
         return dict_to_schema(d, required=True, modifier=expand_system_vars)
 
-    def create_instance(self, plugin, **instance_kwargs):
+    def create_instance(self, plugin_name: str, **instance_kwargs) -> object:
```

Comment:
```suggestion
    def create_instance(self, plugin_name: str, **instance_kwargs) -> typing.Any:
```
I think `Any` is slightly better for mypy. Any declares that the return is not typed.